### DISCUSSION TITLE
Update values of single column (continuously) without updating whole table rows

### DISCUSSION THREAD

**Discussion by vishg333** [Comcast/react-data-grid#2579] [Q&A]
Hi guys,
I want to update the data of only single column from datagrid(continuously like live streaming) without affecting/updating other column's data(other column's static data is going to be static) using **Functional component**.

I know there is something called rowRenderer and cellRenderer but I don't know how to use these in my case. If someone has any idea or came across such case, please share the solution. It would be of great help to me.

Thanks in advance 🙂

**Comment by nstepien** [comment]
I'd suggest streaming your data into a `Context` instead, then your `Column.formatter` can listen to changes for that context only.
https://reactjs.org/docs/context.html
